stream_alert/rule_processor/handler.py

```diff
@@ -24,6 +27,10 @@
 from stream_alert.rule_processor.rules_engine import StreamRules
 from stream_alert.rule_processor.sink import StreamSink
 from stream_alert.shared.metrics import Metrics
+from stream_alert.shared.stats import time_me
+
+NUM_WORKERS = 6
```

Comment:
@jacknagz I was going to do away with this and always default to 2x the available CPUs (in lambda this will always == 4). I think it could be v bad to allow users to set this to whatever they 'think' is appropriate